docs/DesignDocs/CustomAttestation.md

```diff
@@ -927,9 +965,24 @@ size_t params_size = sizeof(params);
 oe_claim_t claims = { ... };
 size_t claims_size = ...;
 
+/* Receive the evidence format ids that the verifier supports */
+recv(VERIFIER_SOCKET_FD, evidence_format_ids, evidence_format_id_length, 0);
```

Comment:
I would move this up right after the `oe_register_attester(my_plugin_attester(), &config, config_size);` line, because the `struct my_plugin_attester_opt_params_t params = { ... };` line is plugin specific, so you want to decide on the common plugin before you do the input params.